exercises/practice/http-billion-dollar-question/.docs/instructions.md

```diff
@@ -4,19 +4,20 @@
 
 You have been asked to find the top `x` billionaires by net worth in a given set of countries (number of countries = `N`). To implement that, you have been given access to an API that returns the details of the billionaires by country.
 
-To simplify accessing the API, you can use the [`ims/billionairehub` client](https://lib.ballerina.io/ims/billionairehub/1.0.0/clients/Client) connector to connect to the API which returns details of the billionaires by country.
+To simplify accessing the API, you can use a predefined client connector to connect to the API which returns details of the billionaires by country.
 
 
 ```ballerina
-import ims/billionairehub;
+import ballerina_exercism/http_billion_dollar_question.billionarehub_server as _;
+import ballerina_exercism/http_billion_dollar_question.billionarehub_client as billionarehubClient;
 
-# Client ID and Client Secret to connect to the billionaire API
-configurable string clientId = ?;
-configurable string clientSecret = ?;
+type Billionaire record {
+    string name;
+    float netWorth;
+};
```

Comment:
It seems we can have a closed record.
```suggestion
type Billionaire record {|
    string name;
    float netWorth;
|};
```

Reply:
Will update.

Reply:
Updated. Please check. dff21f3a570e21b4a892f362369d89c88b846425